cfgov/v1/migrations/0233_clean_orphaned_revisions.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.17 on 2023-03-14 13:09
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Page = apps.get_model('wagtailcore', 'Page')
+    PageRevision = apps.get_model('wagtailcore', 'PageRevision')
+
+    for revision_pk, page_pk in PageRevision.objects.values_list('pk', 'page'):
+        try:
+            page = Page.objects.get(pk=page_pk)
+        except Page.DoesNotExist:
+            page = None
+        else:
+            continue
+
+        revision = PageRevision.objects.get(pk=revision_pk)
+        print("Orphaned:", revision_pk, revision.content["url_path"])
```

Comment:
Is there some use of `logger` here we can use instead, to ensure that this gets persisted into our centralized log storage? Additionally, it'd be nice to add `page_pk` to the log statement for posterity.

Reply:
Oh, yes indeed. The `print()` was totally for debugging 😁

Reply:
Replaced with logging in [d534882](https://github.com/cfpb/consumerfinance.gov/pull/7569/commits/d5348825663125f5da39d1f5aaa1a1eb7d682419).